tests/perf/size_check_debug.test

```diff
@@ -1,5 +1,5 @@
 program="../size"
-max_size=1400000
+max_size=1500000
```

Comment:
This jumped up from something like 100k to, now, 1.5MB. So I guess it's triggered something's going wrong, in general.

Nothing to do with this PR though.